### PR TITLE
Add aws-bootstrap-kit

### PR DESCRIPTION
**Pull request by ijemmy**
I created a new category in `Construct Libraries` as it doesn't fit into an existing one.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -76,6 +77,9 @@ This section includes code libraries in various programming languages which vend
 * [cdk-watchful](https://github.com/eladb/cdk-watchful) - Automatic dashboards and alarms for CDK apps.
 * [aws-cdk-billing-alarm](https://github.com/alvyn279/aws-cdk-billing-alarm) - Construct that sets up email alerts for exceeding an amount on your AWS bill.
 
+### Organization setup
+* [aws-bootstrap-kit](https://github.com/awslabs/aws-bootstrap-kit) - Creates a multi-account set-up with AWS Organization, AWS SSO, DNS, and AWS CodePipeline.
```
Comment (kolomied) [inline readme.md:81]: AWS Organization**s**

File: readme.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ The AWS Cloud Development Kit (AWS CDK) is an open-source software development f
   * [Queue](#queue)
   * [CI/CD](#cicd)
   * [Monitoring](#monitoring)
+  * [Organization setup](#organization-setup)
```
Comment (kolomied) [inline readme.md:18]: Should we name it just `Organizations` to be a bit more generic?
Please also add the new item to the end of the list - a general approach we follow

File: readme.md (hunk 1 above)
Comment (ijemmy) [reply readme.md:81]: Good catch. Will fix this.

File: readme.md (hunk 2 above)
Comment (ijemmy) [reply readme.md:18]: * `Organizations` doesn't convey much meaning here (apart from we're using up AWS Organizations).  What about `Multi-accounts setup`. Would that make more sense? 
* Will move this to the end of the list.

Comment (ijemmy) [reply readme.md]: Have been pondering about this, I think `Multi-accounts setup` make most sense. People are more likely to search for this term. Put this in the new version.

**Comment by ijemmy** [comment]
@kolomied  Fixed the comments above.

**Review by kolomied — APPROVED** [review]

**Comment by kolomied** [comment]
@ijemmy Please add "Mentioned in awesome" badge to your repo with the link to this list - [example](https://github.com/kolomied/cdk-stepfunctions-patterns).

Thank you for the contribution! A nice and useful construct 👍

**Comment by ijemmy** [comment]
Thanks. Sure, I will add the badge in.

**Pull request merged**